net/mwan3/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2014 OpenWrt.org
+# Copyright (C) 2006-2017 OpenWrt.org
```

Comment:
As you are not under contract with OpenWrt, you should not update the OpenWrt copyright